apps/spruce/src/pages/version/Tabs/TimeChart/TimeChartGraph/index.tsx

```diff
@@ -0,0 +1,59 @@
+import { Spinner } from "@leafygreen-ui/loading-indicator";
+import { palette } from "@leafygreen-ui/palette";
+import Chart from "react-google-charts";
+import { GanttChartData } from "../types";
+
+const CHART_ROW_HEIGHT_IN_PIXELS = 42;
+const MINIMUM_CHART_HEIGHT_IN_PIXELS = 400;
+
+interface Props {
+  data: GanttChartData;
+  loading: boolean;
+}
+
+const TimeChartGraph: React.FC<Props> = ({ data, loading }) => {
+  if (loading || !data || data.length === 1) {
+    return (
+      <div
+        style={{
+          display: "flex",
+          justifyContent: "center",
+          alignItems: "center",
+          height: `${MINIMUM_CHART_HEIGHT_IN_PIXELS}px`,
+        }}
+      >
+        {loading ? <Spinner /> : <p>No data available</p>}
+      </div>
+    );
+  }
+
+  return (
+    <Chart
+      chartType="Gantt"
+      data={data}
+      height={Math.max(
+        data.length * CHART_ROW_HEIGHT_IN_PIXELS,
+        MINIMUM_CHART_HEIGHT_IN_PIXELS,
+      )}
+      options={{
```

Comment:
The time labels on the x-axis don't mean anything since they don't have any associated dates.  12pm and 12am show up twice in this example but they aren't associated with any particular date.
![image](https://github.com/user-attachments/assets/56abd3be-c900-42b0-ac66-807216648588)